bring_api/bring.py

```diff
@@ -38,8 +43,33 @@ def __init__(
 
         self.mail = mail
         self.password = password
-        self.uuid = ""
         self.public_uuid = ""
+        self.userlistsettings: dict[str, dict[str, str]] = {}
+        self.user_locale = "de-CH"
+        self.supported_locales = [

```

Comment:
I would really put this list in a `const.py` file as well as the default `user_locale`.